ledger/nomos-ledger/src/lib.rs

```diff
@@ -392,12 +314,12 @@ impl LedgerState {
         self.nullifiers.contains(nullifier)
     }
 
-    fn update_nonce(self, nullifier: Nullifier, slot: Slot) -> Self {
+    fn update_nonce(self, contrib: [u8; 32], slot: Slot) -> Self {
```

Comment:
Should the `[u8; 32]` by its own type? maybe an alias?

Reply:
Not sure, it's really just a bunch of bytes and we don't have a separate nonce type for instance. Not to say that it's not a good idea, but I'd like to see the full new structure before this